plugins/gestures.koplugin/defaults.lua

```diff
@@ -76,6 +84,14 @@ return {
         hold_top_right_corner = nil,
         hold_bottom_left_corner = nil,
         hold_bottom_right_corner = nil,
+        one_finger_swipe_east = nil,
+        one_finger_swipe_west = nil,
+        one_finger_swipe_south = {full_refresh = true,},
+        one_finger_swipe_north = {full_refresh = true,},
+        one_finger_swipe_northeast = nil,
+        one_finger_swipe_northwest = nil,
+        one_finger_swipe_southeast = nil,
+        one_finger_swipe_southwest = nil,
```

Comment:
Speaking of the full refresh, that's the other way around: it was intended to be bound to *any* diagonal swipe.

Reply:
Ok, I'll switch the defaults. I guess I got confused by [this](https://github.com/koreader/koreader/issues/6741#issuecomment-771851683). But users can configure it later anyway. We just want to make it the least disruptive possible.